Sources/Splash/Theming/Font.swift

```diff
@@ -31,6 +31,12 @@ public struct Font {
         resource = .system
         self.size = size
     }
+    
+    /// Initialize an instance with an existing, custom, font.
+    public init(font: Loaded) {
```

Comment:
I think here we can drop the external parameter label, so rather than:
```swift
Font(font: nativeFont)
```
It would be:
```swift
Font(nativeFont)
```
Or what do you think @terhechte?

Reply:
Yeah, I like it. Makes it clearer. If you're fine with this change, I'd check out the repo and create a proper PR, seems CI failed (I did the editing right on Github, so I never tried to compile it ;)

Reply:
Sounds great! 👍